man/man8/zpool-scrub.8

```diff
@@ -91,6 +96,32 @@ again.
 .It Fl w
 Wait until scrub has completed before returning.
 .El
+.Sh EXAMPLES
+.Bl -tag -width "Exam"
+.It Sy Example 1 : Status of pool with ongoing Scrub:
+Output:
+.Bd -literal -compact -offset Ds
+.No # Nm zpool Cm status
+  ...
+  scan: scrub in progress since Sun Jul 25 16:07:49 2021
+        403M scanned at 100M/s, 68.4M issued at 10.0M/s, 405M total
+        0B repaired, 16.91% done, 00:00:04 to go
+  ...
+.Ed
+, where:
+.Bl -dash -offset indent
+.It
+Metadata for 403M of data was scanned at 100M/s:
+.Bd -literal -compact -offset Ds
+.No 403M scanned at 100M/s
+.Ed
+.It
+68.4M of data read issued sequentially at 10.0M/s:
+.Bd -literal -compact -offset Ds
+.No 68.4M issued at 10.0M/s
+.Ed
```

Comment:
I like the idea of clearly explaining how to interpret these values with an example.  Perhaps it'd be more user friendly to do it with a paragraph of text like so?  I'm open to whatever formatting works best for this.

```
     Example 1: Status of pool with ongoing Scrub:
           Output:
                 # zpool status
                   ...
                   scan: scrub in progress since Sun Jul 25 16:07:49 2021
                         403M scanned at 100M/s, 68.4M issued at 10.0M/s, 405M total
                         0B repaired, 16.91% done, 00:00:04 to go
                   ...
           Where:

                 -   Metadata which references 403M of file data has been
                     scanned at 100M/s, and 68.4M of that file data has been
                     scrubbed sequentially at 10.0M/s.
```